css/BEM-style.md

```diff
@@ -0,0 +1,223 @@
+        <div class="hero-box__text">
+            <h1 class="primary-heading">I love BEM Styling</h1>
+        </div>
+        <div class="hero-box__buttons">
+            <a href="#"class="btn btn--active">I know!</a>
+            <a href="#"class="btn btn--ghosted">I surely know!</a>
+        </div>
+    </div>
+</section>
+```
+
+You can now pretty much style all the components the way you like, just by using the names of the classes only (remember BEM was all about making specificity low and avoiding conflicts).
+>If you are using SaSS , BEM becomes more powerful because now you can use the Sass nesting feature for all the classes.
+```
+.hero-box{
+    height: 100vh;
+    background:linear-gradient(to bottom right,#000,transparent),
+    url("https://images.unsplash.com/photo-1580554996018-ff8b408fc162?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=750&q=80");
+     background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    display:flex;
+    flex-direction:column;
+    justify-content: center;
+    align-items: flex-start;
+    
+    &__text{
+       margin:2rem;
+    } 
+    &__buttons{
+        margin:0.5rem;   
+    }  
+}
+.btn{
+    text-decoration: none;
+    border:1px solid saddlebrown;
+    padding:1rem 2rem;
+    color:white;
+    border-radius:113rem;
+    font-size:2rem;
+    text-align: center;
+    display:inline-block;
+    margin:2rem;
+    &--active{
+         background:saddlebrown;
+         &:hover{
+             background-color:rgb(201, 93, 17);
+         }
+    }
+    &--ghosted{
+        background:transparent;
+        &:hover{
+            background-color:rgb(201, 93, 17);
+        }
+    }
+}
+.primary-heading{
+    font-size:6rem;
+}
+```
+
+**3.How BEM is useful on larger codebases and scaling and writing maintainnable code:** 
+
+What if I told you to add a new button with some animation on it in the hero-box section? What other information will you need? Well nothing, since all necessary information is already present with you now and you know exactly where this component must lie and how you will name the classes on it. 
+This process of naming modifiers,elements and blocks might seem tedious at first, but as your project will grow larger and larger you'll realise how easy it has become to find and change the styles on a component or add styles to new components created. It made your HTML and CSS much more readable since the names of classes now make sense. You no longer need to worry about the specificity now and find out which particular class/id/tag is conflicting your newly written style.
+
+
+### Independent Practice
+
+Since you can use BEM pretty much anywhere , how about refactoring your previously written code for a project (very old projects). This will show you how your previously named classes, now makes it difficult for you to understand and maintain CSS code, but using BEM now will no longer create that issue for you. 
```

Comment:
Can you split this into more sentences so you dont need parens or quite so many commas?